### PR TITLE
Add ESLint on TravisCI (fixes #5)

### LINKED ISSUE #5: Add lint/style check in Travis

### PR DESCRIPTION
**Pull request by leplatrem**
UglifyJS reported errors in lib/, so I took some time to introduce ESLint.

    Dropping unused variable results [./~/kinto/lib/api.js:186,0]
    Side effects in initialization of unused variable status [./~/kinto/lib/api.js:243,0]
    Side effects in initialization of unused variable headers [./~/kinto/lib/api.js:245,0]
    Side effects in initialization of unused variable _errorsJs2 [./~/kinto/lib/api.js:21,0]
    Side effects in initialization of unused variable deleted [./~/kinto/lib/collection.js:477,0]
    Side effects in initialization of unused variable request [./~/kinto/lib/adapters/IDB.js:190,0]
    Side effects in initialization of unused variable request [./~/kinto/lib/adapters/IDB.js:316,0]

@n1k0  thoughts?

### CODE REVIEW

**Comment by leplatrem** [comment]
I added the test folder, but gave up on linting ``comma-dangle``. Too many errors.

File: test/adapters/localStorage_test.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe("adapter.LocalStorage", () => {
     describe("#create", () => {
       it("should add created key to the key list", () => {
         return db.create({id: 1})
-          then(_ => expect(db.keys).eql([1]));
+          .then(_ => expect(db.keys).eql([1]));
```
Comment (leplatrem) [inline test/adapters/localStorage_test.js]: /me still wonders how it could work.

File: test/integration_test.js [hunk 2]
```diff
@@ -96,7 +95,7 @@ describe("Integration tests", () => {
     describe("Settings", () => {
       it("should retrieve server settings", () => {
         return tasks.sync().then(_ => tasks.api.serverSettings)
-          to.eventualy.include.keys("cliquet.batch_max_requests");
+          .to.eventualy.include.keys("cliquet.batch_max_requests");
```
Comment (leplatrem) [inline test/integration_test.js]: Look at this nasty one :)

Comment (n1k0) [inline test/integration_test.js]: This is crazy.

File: test/adapters/localStorage_test.js (hunk 1 above)
Comment (n1k0) [inline test/adapters/localStorage_test.js]: /me too.

**Comment by n1k0** [comment]
r+

**Pull request merged**